app/pages/lab-pages-editor/helpers/createTask.js

```diff
@@ -0,0 +1,17 @@
+/*
+Creates an empty/placeholder Task that's ready to be inserted into a Workflow.
+NOTE: the Task Key isn't handled by this function. Whatever's calling this
+function needs to assign the appropriate Task Key to this Task, and then add it
+to the Workflow.
+ */
+
+export default function createTask(taskType = 'text') {
+  // TODO
+  // Placeholder only
+  return {
+    help: '',
+    instruction: 'This is an example Text Task.',
+    required: false,
+    type: 'text'
+  };
```

Comment:
This function should return `TextTask.getDefaultTask()`

https://github.com/zooniverse/Panoptes-Front-End/blob/d9834aa0b47b7f13053c59bf7dd133c162370923/app/classifier/tasks/text/index.jsx#L148-L153

Reply:
```js
import tasks from '../../classifier/tasks';

export default function createTask(type) {
  return tasks[type]?.getDefaultTask();
}
```

Reply:
👍